src/components/Book.js

```diff
@@ -1,16 +1,46 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
 
 const Book = ({ book, handleRemoveBook }) => {
   const handleBookRemove = () => {
     handleRemoveBook(book);
   };
+  const percentage = 66;
   return (
-    <tr key={book.id} id={book.id}>
-      <td>{book.title}</td>
-      <td>{book.category}</td>
-      <td><button type="button" onClick={handleBookRemove} className="btn btn-danger">Delete</button></td>
-    </tr>
+    <div className="card">
+      <div className="row card-inside">
+        <div className="col-12 col-sm-3 col-md-4">
+          <p className="gender-book">{book.category}</p>
+          <p className="title-book">{book.title}</p>
+          <p className="author-book">JRR Tolkien</p>
+          <div className="row">
+            <div className="col-12 col-lg-5">
+              <a className="book-option" href="/">Comments</a>
+            </div>
+            <div className="col-12 col-lg-4">
+              <a className="book-option" href="/">Remove</a>
+            </div>
+            <div className="col-12 col-lg-3">
+              <a className="book-option" href="/">Edit</a>
+            </div>
```

Comment:
- [ ] title, author, comments, remove and edit should have the same font, this is a free[ google font ](https://fonts.google.com/specimen/Roboto+Slab?query=Roboto)that can be applied.
- [ ] Inspect how these elements are arranged and fix the styling accordingly.